go/theatre/approval_test.go

```diff
@@ -0,0 +1,78 @@
+package theatre_test
```

Comment:
Really pleased to see you write an approval testing tool and not just having the string to be compared included in the test source code. You've clearly put some effort in to make the code general and re-usable. Why not go the whole hog and create a reusable package? And/or even add as a project under https://github.com/approvals? Llewellyn (https://github.com/isidore) is very friendly and often offers to pair with people on this kind of thing if you'd like a hand.

Reply:
Good recommendation. I am hesitating because it's still not clear to me how exactly these approval tests should look like (what the 'core pattern' is). Comparing the different approaches, some things are provided, others are not. So I took an approach on how I would expect it to behave.
Yet, there's already the Twitter exchange - see final comment.